eng/PatchConfig.props

```diff
@@ -15,14 +15,13 @@ Directory.Build.props checks this property using the following condition:
 
   <PropertyGroup Condition=" '$(VersionPrefix)' == '3.0.1' ">
     <PackagesInPatch>
-      Microsoft.Net.Http.Headers;
-      Microsoft.AspNetCore.CookiePolicy;
-      Microsoft.AspNetCore.DataProtection.EntityFrameworkCore;
-      @microsoft/signalr;
-      Microsoft.Net.Http.Headers;
-      Microsoft.AspNetCore.Http.Abstractions;
-      Microsoft.AspNetCore.Http.Features;
-      Microsoft.AspNetCore.CookiePolicy;
+      Microsoft.AspNetCore.Blazor;
+      Microsoft.AspNetCore.Blazor.Build;
+      Microsoft.AspNetCore.Blazor.DataAnnotations.Validation;
+      Microsoft.AspNetCore.Blazor.DevServer;
+      Microsoft.AspNetCore.Blazor.HttpClient;
+      Microsoft.AspNetCore.Blazor.Server;
+      Microsoft.AspNetCore.Blazor.Templates;
```

Comment:
How does this file matter in a non-servicing build? Even if it mattered, the `Condition` would prevent use of this property group.

Reply:
It doesn't, but it seemed nicer to have it represent the stuff this branch builds. Doesn't matter much either way, I suppose

Reply:
What this branch builds should have nothing to do with PatchConfig.props. These changes further confuse by seeming to change what shipped in 3.1.2. Please revert changes to this file.

Reply:
Or just delete this file.

Reply:
I undid the changes